src/main/java/seedu/sudohr/logic/commands/leave/ListLeavesByEmployeeCommand.java

```diff
@@ -0,0 +1,61 @@
+package seedu.sudohr.logic.commands.leave;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.sudohr.logic.parser.CliSyntax.PREFIX_EMPLOYEE;
+
+import seedu.sudohr.logic.commands.Command;
+import seedu.sudohr.logic.commands.CommandResult;
+import seedu.sudohr.logic.commands.exceptions.CommandException;
+import seedu.sudohr.model.Model;
+import seedu.sudohr.model.employee.Employee;
+import seedu.sudohr.model.employee.Id;
+
+/**
+ * Lists leaves by a specified employee.
+ */
+public class ListLeavesByEmployeeCommand extends Command {
+    public static final String COMMAND_WORD = "llbe";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Lists the leaves taken by the specified employee"
+            + "Parameters: EMPLOYEE_ID \n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_EMPLOYEE + "101";
+
+    public static final String MESSAGE_SUCCESS = "Listed all leaves by employee %1$s";
+
+    public static final String MESSAGE_EMPLOYEE_NOT_FOUND = "No employee exists with the given employee id.";
```

Comment:
Not your issue but I just recalled that we should be moving this message to commons

Reply:
yep